tests/status/worktree.c

```diff
@@ -1299,7 +1299,8 @@ void test_status_worktree__at_head_parent(void)
   cl_git_rewritefile("empty_standard_repo/file2", "pyng");
```

Comment:
Same regaring the commit message. Furthermore, it feels like this should be squashed in to the initial commit where the test has been created with `git_status_list_new` taking the option?